Capacity design doc refinement (#807)
LibertyDSNP/frequency: designdocs/capacity.md

```rust

pub enum Event<T: Config> {
  /// Tokens have been staked to the Frequency network.
  Staked {
    /// The token account that staked tokens to the network.
    account: T::AccountId,
    /// The MSA that a token account targeted to receive Capacity based on this staking amount.
    target: MessageSourceId,
    /// An amount that was staked.
    amount: BalanceOf<T>
  },

>   /// A token account has unstaked the Frequency network.

Reviewer: ```suggestion
  /// A token account has unstaked token from the Frequency network.
```

```rust

pub struct StakingAccountDetails<Balance, BlockNumber> {

>   /// The amount a Staker has staked, minus the sum of all tokens in `unlocking`.

Reviewer: This may be more clear:
```suggestion
  /// The amount a Staker has staked, not counting tokens in `unlocking`.
```
